src/Uno.UWP/Storage/Pickers/Internal/FileOpenPickerActivity.Android.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Android.App;
+using Android.Content.PM;
+using Android.OS;
+using Android.Provider;
+using Android.Content;
+using AndroidUri = Android.Net.Uri;
+
+namespace Uno.Storage.Pickers.Internal
+{
+	[Activity(ConfigurationChanges = ConfigChanges.Orientation | ConfigChanges.ScreenSize)]
+	internal class FileOpenPickerActivity : Activity
+	{
+		internal static event EventHandler<List<AndroidUri>> FilePicked;
+
+		private bool _multiple;
+		
+		protected override void OnCreate(Bundle savedInstanceState)
+		{
+			base.OnCreate(savedInstanceState);
+
+			var caller = base.Intent.Extras;
+			_multiple = caller.GetBoolean(Intent.ExtraAllowMultiple, false);
+
+			// startpicker
+			var intent = CreateIntent(_multiple,
+				caller.GetStringArray(Intent.ExtraMimeTypes).ToList(),
+				caller.GetString(DocumentsContract.ExtraInitialUri, "")
+				);
+			// StartActivityForResult(Android.Content.Intent.CreateChooser(intent, "Select file"),10);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5260159)